### ISSUE TITLE
[Storybook] Storybook doesn't seem to have accessibility support

### ISSUE THREAD

**Issue by camilasb** [storybookjs/storybook#8220]
Steps:

Open the IoT Storybook
Try to navigate on the left pane using the keyboard
Actual Result: User is not able to do it properly
Expected Result: User should be able to navigate thru storybook properly using the keyboard
![gif](https://user-images.githubusercontent.com/20360162/65781763-9b2bfe00-e122-11e9-976e-eb1d9ac967dc.gif)

Graphite (from Maximo product) team depends on accessibility to be able to put the product in the market, so we need this support.

**Comment by shilman** [comment]
Closing as duplicate to #7833

Contributions welcome!
